build.gradle

```diff
@@ -26,7 +26,7 @@ ext {
   sizeofVersion = '0.3.0'
 
   // Clustered
-  terracottaPlatformVersion = '5.0.6.beta6'
+  terracottaPlatformVersion = '5.0.6.beta7'
```

Comment:
- the first upgrade was not necessary: I did it only to match the latest state as yesterday.
- I could split in 2 PR. But what would be the benefits ? It would take extra time to build, you'll need to review / merge 2 consecutive PRs...